src/picomsg/protobuf/config_conn/config_conn_api.proto

```diff
@@ -87,6 +87,7 @@ message AddSystemEnterpriseRequest {
 	 * The private IP assigned to the terminator.
 	 * Used as the nexthop for the installed routes
 	 */
+	string neigh_sub_interface = 7;
```

Comment:
Update comment at beginning of message (now maps to `neigh_sub_interface` as well)